provider/cmd/pulumi-resource-cloud-toolkit-aws/databases/mysqlArgs.ts

```diff
@@ -0,0 +1,225 @@
+import defaultsDeep from "lodash.defaultsdeep";
+import * as pulumi from "@pulumi/pulumi";
+
+/**
+* Arguments to deploy a database instance
+*/
+export interface MysqlArgs {
+/**
+ * Instance type to run the database instance
+ */
+  instance?: pulumi.Input<string>;
+
+/**
+ * Version for database instance
+*/
+  version: pulumi.Input<string>;
```

Comment:
Shouldn't this be `MysqlVersions` instead of an Input?

Reply:
Changed to MysqlVersions